docs/checkit.js

```diff
@@ -84,7 +84,12 @@ function CheckIt() {
     
     // We have to connect to Firebase before we can access it.
     // WHEN PAGE LOADS
-    this.initFirebase();
+    if (this.mode === 'firebase') {
+        this.initFirebase();
+    }
+    else {
```

Comment:
I'd probably explicitly check for `'localStorage`' and throw otherwise. That protects you against typos like `new Checkit('firebsae');`.
```
switch (this.mode) {
  case 'firebase': this.initFirebase(); break;
  case 'localStorage': this.initLocalStorage(); break;
  default: throw new Error("Unknown storage mode: '" + this.mode + "'");
}
```